documentation/2.0/content/release-notes.md

```diff
@@ -35,3 +35,9 @@ java.lang.IllegalArgumentException: In production mode, it's not allowed to set
 **ISSUE**: For WLS versions prior to 14.1.1, there is a problem setting the `GroupMemberOf` attribute in the `topology/Security/User` folder. The value is not persisted correctly, and the assignment will not be present when the domain is started.
 
 **ACTION**: Contact Oracle Support to obtain the patch for bug number 30319071 for your WebLogic Server version before running the tool.
+
+#### Problems setting `RotateLogOnStartup` attribute
+
+**ISSUE**: For WLS versions up to and including 14.1.2, there is a problem setting the `RotateLogOnStartup` attribute in various log file folders. The value is not persisted correctly, and the assignment will not be present when the domain is started.
```

Comment:
I don't think we should mention 14.1.2.  The numbering for the next WebLogic release is still undecided (and we shouldn't be discussing future WLS releases anyway)...